core/base/base.go

```diff
@@ -21,17 +21,20 @@ type Base struct {
 	Channel Channel
 }
 
+// Empty is an empty base.
+var Empty = Base{}
```

Comment:
Not sure I like this as an exported global variable.

It's only a matter of time before someone passes it by reference and ends up modifying it.

Reply:
OK, I'll remove it.